acceptance/tests/vault/vault_namespaces_test.go

```diff
@@ -20,10 +20,274 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// TestVault_VaultNamespace_AdditionalConfig installs Vault, configures a Vault namespace, and then bootstraps it
+// with secrets, policies, and Kube Auth Method.
+// It then configures Consul to use vault as the backend and checks that it works
+// with the vault namespace. Namespace is added in this via global.secretsBackend.vault.connectCA.additionalConfig
+func TestVault_VaultNamespace_AdditionalConfig(t *testing.T) {
```

Comment:
Instead of copying the original test, could we refactor it into a helper that allows us to decide which form of config to use, and then run it once per case? That would reduce the risk of drift between the two tests in the future and make the difference between them more obvious.